admin-ui/plugins/auth-server/components/AuthN/AuthNForm.js

```diff
@@ -56,9 +55,9 @@ function AuthNForm({ item, handleSubmit }) {
     setModal(!modal)
   }
 
-  const submitForm = () => {
+  const submitForm = (userMessage) => {
     toggle()
-    handleSubmit(formik.values)
+    handleSubmit(formik.values, userMessage)
   }
```

Comment:
Submitting a form value without normalizing its data type is bad because it violates the principle of data integrity by passing a string value (`"true"` or `"false"`) to a consumer that expects a strict boolean, which can lead to incorrect logical evaluations or API validation failures.